### DISCUSSION TITLE
"Charm" Skill Trigger - "Int" or "Cha"?

### DISCUSSION THREAD

**Discussion by msprijatelj** [massif-press/compcon#2134] [Archive]
This is a mostly trivial question, but I was wondering if the "Charm" skill trigger should be classified as "Charisma" instead of "Intelligence" for the purposes of organizing the Skill Trigger selection page? My personal opinion is that since Charm focuses on persuasion, it should be grouped with the Charisma group, feeling like one way a pilot can "talk, lead, change minds, make connections, and requisition resources". Thoughts?

**Comment by super-salad** [comment]
I agree that it should be "Charisma". It's usually classified as such in other systems that use character ability scores (looking at you, D&D). Are  you working on an alternate grouping feature?

**Reply by msprijatelj** [reply]
Nope! The existing "Str/Dex/Int/Cha" distinction is fine by me. This would be a one-line change in the lancer-data repo. I just noticed that the Charisma skills seemed a little thin, while the Intelligence skills seemed more populous, and Charm itself seemed out-of-place.
